historian-server/src/main/java/com/hurence/webapiservice/historian/impl/SolrHistorianServiceImpl.java

```diff
@@ -659,7 +665,8 @@ private JsonObject extractTimeSeriesThenBuildResponse(List<JsonObject> chunks, M
     private JsonObject buildTimeSeriesResponse(MultiTimeSeriesExtracter timeSeriesExtracter) {
         return new JsonObject()
                 .put(TOTAL_POINTS, timeSeriesExtracter.pointCount())
-                .put(TIMESERIES, timeSeriesExtracter.getTimeSeries());
+                .put(TIMESERIES, timeSeriesExtracter.getTimeSeries())
+                .put(AGGREGATION, timeSeriesExtracter.getAggregation());
```

Comment:
Here think about the what is the response like. I am not sur about separating TIMESERIES and AGGREGATION

Reply:
yes, it was an error, i did change this in second commit